Cargo.toml

```diff
@@ -11,7 +11,7 @@ categories = ["wasm", "gui", "web-programming"]
 
 [dependencies]
 log = "0.4"
-dioxus = { version = "0.5", default-features = false, features = ["launch", "macro", "html"] }
+dioxus = { version = "0.6", default-features = false, features = ["launch", "macro", "html", "signals"] }
```

Comment:
Haven't tried running it yet, but, do we really need the `signals` feature? 👀

Reply:
I don't know enough about the library or dioxus to be certain, but it doesn't compile without it. 

got the suggestion from here: https://discord.com/channels/899851952891002890/1317566926175735849/1317578656167493685

Reply:
I see! All good!